app/controllers/application_controller.rb

```diff
@@ -1,2 +1,11 @@
 class ApplicationController < ActionController::API
+  include ActionController::Serialization
+
+  def start_id
+    params[:start_id].to_i
+  end
+
+  def count
+    params[:count] || default_count
+  end
```

Comment:
That's true for right now.  But I'm thinking pagination will be a frequent feature of our application.  And most controllers will need this.  If it's confusing for right now though happy to move it back.  It's a trivial change.